File: app/src/main/java/kfzkennzeichen/gisbertamm/de/kfz_kennzeichen/OnSearchCompletedListener.java
package kfzkennzeichen.gisbertamm.de.kfz_kennzeichen;

import kfzkennzeichen.gisbertamm.de.kfz_kennzeichen.persistence.SavedEntry;

public interface OnSearchCompletedListener {
    public void onSearchCompleted(SavedEntry entry, String code);
}

File: app/src/main/java/kfzkennzeichen/gisbertamm/de/kfz_kennzeichen/OnSearchCompletedListener.kt
package kfzkennzeichen.gisbertamm.de.kfz_kennzeichen

import kfzkennzeichen.gisbertamm.de.kfz_kennzeichen.persistence.SavedEntry

interface OnSearchCompletedListener {
    fun onSearchCompleted(entry: SavedEntry?, code: String?)
    fun onStatisticsCompleted(statistics: Map<String, Int>?)
}